openmetadata-mcp/src/main/java/org/openmetadata/mcp/MCPStreamableHttpServlet.java

```diff
@@ -296,6 +295,15 @@ private void startSSEStreamForRequest(
       String sessionId)
       throws IOException {
 
+    // Check connection limit
+    if (!sseConnectionSemaphore.tryAcquire()) {
```

Comment:
same here about the semaphore number. Is 50 always going to be a safe number? What about orgs with bunch of users?

Maybe I need to understand better what these numbers mean

Reply:
It's released at the end of this functions !